File: BackEndAPI.Java/DiChoThue/src/main/java/com/example/DiChoThue/Entities/DiaChi.java
package com.example.DiChoThue.Entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "DIA_CHI")
public class DiaChi {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int ma_dia_chi;
    private String ten_dia_chi;
    private int toa_do_dong;
    private int toa_do_bac;
    private int loai_vung;
}

File: BackEndAPI/Data/Entities/DiaChi.cs
using BackEndAPI.Data.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BackEndAPI.Data.Entities
{
    public class DiaChi
    {
        public int MaDiaChi { get; set; }
        public string TenDiaChi { get; set; }
        public int ToaDoDong { get; set; }
        public int ToaDoTay { get; set; }
        public LoaiVung LoaiVung { get; set; }
    }
}
